common/djangoapps/student/views.py

```diff
@@ -1754,6 +1780,12 @@ def create_account_with_params(request, params):
             }
         )
 
+    if settings.FEATURES.get('ENABLE_SAILTHRU'):
```

Comment:
I think I mentioned this to @brianfohl, but I'd like to see this done via a ConfigurationModel, rather than a flag in `FEATURES`.